### PR TITLE
Remove the bitflux tearout judder

### LINKED ISSUE #528: Moving a tearout card over Bitflux causes judder
Moving the tearout card around the place is fine and smooth, but if it goes over the Bitflux chart, it starts juddering a bit.

Edit: Experimentation -- Seems to be related to number of days loaded on the chart

### PR DESCRIPTION
**Pull request by WPFerg**
Closes #528

### CODE REVIEW

**Comment by bjedrzejewski** [comment]
@WPFerg can you make it merge-able?

**Comment by WPFerg** [comment]
@bjedrzejewski done :)

File: src/main/main-controller.js [hunk 1]
```diff
@@ -2,8 +2,30 @@
...
+
+            this.tearingOut = false;
+            this.watch();
+        }
+
+        watch() {
+            this.$scope.$on('tearoutStart', () => {
+                this.$timeout(() => {
+                    this.tearingOut = true;
+                });
```
Comment (owennw) [inline src/main/main-controller.js]: Why do you need the `$timeout`s?

Comment (WPFerg) [inline src/main/main-controller.js]: It doesn't apply the class otherwise

**Pull request merged**